test/SlowTests/Issues/RavenDB_20079.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Linq;
+using FastTests;
+using Raven.Client.Documents.Indexes;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues;
+
+public class RavenDB_20079 : RavenTestBase
+{
+    public RavenDB_20079(ITestOutputHelper output) : base(output)
+    {
+    }
+    
+    [Fact]
+    public void CanCreateIndexWithDateTime()
+    {
+        using (var store = GetDocumentStore())
+        {
+            var index = new TestIndex();
+            index.Execute(store);
```

Comment:
Can you add an empty doc and query assert as well?
like `Where(x => x.DefaultDateTime == default)` to check if server evaluated it properly?

Reply:
Added